codalab_service.py

```diff
@@ -270,6 +271,20 @@ def has_callable_default(self):
         default=2048,
         help='Default memory (in MB) for each worker in the AWS Batch Worker Manager',
     ),
+    CodalabArg(
```

Comment:
I added these environment variables because of a known Docker compose issue: https://github.com/docker/compose/issues/3927. Without these the build will fail intermittently.